package.json

```diff
@@ -1,63 +1,49 @@
 {
 	"name": "@monogram/eslint-config",
-	"version": "0.1.2",
+	"version": "0.2.0",
 	"description": "ESLint configs used at Monogram",
 	"keywords": [
 		"eslint",
-		"eslint-config-airbnb",
-		"eslint-config-next"
+		"eslintconfig",
+		"eslint-config"
 	],
 	"license": "MIT",
 	"scripts": {
-		"update-local": "pnpm build && pnpm install-locally",
-		"install-locally": "npm install -g",
-		"prepublish": "pnpm build && pnpm release",
-		"build": "tsc --project tsconfig.json",
-		"prestart": "pnpm build",
-		"start": "ts-node src/index.ts",
-		"release": "standard-version",
-		"postinstall": "pnpm build"
-	},
-	"main": "dist/index.js",
-	"bin": {
-		"eslint-config": "dist/index.js"
-	},
-	"files": [
-		"dist"
-	],
```

Comment:
please keep `files`

Reply:
The reason I moved "files" in package.json to values in `.npmignore` is because with the "files" option, we would have to remember to to update this every time to reflect the current environments that we have. When we do it how I did using `.npmignore`, this isn't the case.